app/views/2017/shared/_header.html.erb

```diff
@@ -15,8 +15,15 @@
       <li class="nav-link"><%= link_to t("header.search"),  "/search" %></li>
       <li class="nav-link"><%= link_to t("header.about"),   "/about" %></li>
     </ul>
+    <% unless I18n.locale == I18n.default_locale %>
+      <ul>
+        <% @locale = LocaleService.find(locale: nil, lang_code: I18n.locale) %>
+        <li class="nav-link"><%= link_to t('header.locale_articles'), "languages/#{@locale.canonical}" %></li>
+      </ul>
+    <% end %>
   </nav>
 
+
```

Comment:
Was this intentional? I'm not sure we need it.

Reply:
Not at all.